lib/packet.mli

```diff
@@ -285,17 +285,27 @@ type q_class = Q_IN | Q_CS | Q_CH | Q_HS | Q_NONE | Q_ANY_CLS
 val q_class_to_string : q_class -> string
 val string_to_q_class : string -> q_class option
 
-(** mDNS unicast response bit. *)
-type q_unicast = QM | QU
+(** This bit is unused in normal DNS, but in mDNS a value
+    of 0 means that multicast responses are desired,
+    while a value of 1 is a request for unicast responses. *)
```

Comment:
This comment is a bit confusing because I believe that normal DNS uses the 0 value as a top bit zero in what is otherwise the query class. The comment also talks about bits but then the constructors are named.

You can also attach comments to individual constructors by putting them at the end of the constructor line (or on the next line).